apps/web/src/components/Layout/Nav/DesktopNav.tsx

```diff
@@ -141,6 +153,9 @@ function DesktopNav({ color }: DesktopNavProps) {
         className={`inline-flex items-center font-mono text-xl ${
           color === 'black' ? 'text-black' : 'text-white'
         }`}
+        onClick={() =>
+          logEventWithContext('bridge', ActionType.click, { componentType: ComponentType.link })
```

Comment:
I think linter's gonna complain about this, it's annoying bug needs to be a function above wrapped in a useCallback